lzero/entry/train_muzero.py

```diff
@@ -161,13 +169,28 @@ def train_muzero(
 
         # Evaluate policy performance.
         if evaluator.should_eval(learner.train_iter):
+            if create_cfg.policy.type in ["muzero_rnn_full_obs"] or cfg.policy.model.model_type in ["conv_context"]:
```

Comment:
implement this part into `_reset_eval` method of policy

Reply:
done